Documentation/operations/system_requirements.rst

```diff
@@ -82,7 +82,7 @@ RancherOS_                 >= 1.5.5
 .. _RancherOS: https://rancher.com/rancher-os/
 
 .. [#centos_foot] CentOS 7 requires a third-party kernel provided by `ElRepo <http://elrepo.org/tiki/tiki-index.php>`_
-    whereas CentOS 8 ships with a supported kernel.
+    whereas CentOS 8 ships with a supported kernel. Cilium features that require large BPF instruction set may not work with CentOS 7 because bpftool shipped with Cilium is not compatible with glibc version on CentOS 7.
```

Comment:
```suggestion
    whereas CentOS 8 ships with a supported kernel. Cilium features that require large BPF programs (eg. VTEP integration) may not work on CentOS 7, as the ``bpftool` version shipped with Cilium can't probe for ``HAVE_LARGE_INSN_LIMIT`` support due to an incompatibility with the ``glibc`` version on CentOS 7.
```

Does that sound correct @vincentmli ? I'd rather not say "large BPF **instruction set**" when we actually mean the program size.

Reply:
got it